src/App.js

```diff
@@ -1,24 +1,29 @@
 // import React, {useState} from 'react';
-import { Switch, Route} from 'react-router-dom';
+import {Switch, Route, Redirect} from 'react-router-dom';
 import './App.css';
 import TopMenu from "./Pages/TopMenu";
 import HomePage from "./Pages/HomePage";
 import LoginPage from "./Pages/LoginPage";
 import BlogPosts from "./Pages/BlogPosts";
 import BlogView from "./Pages/BlogView";
+import {useState} from "react";
 
```

Comment:
Tevens dien je een PrivateRoute.js component te maken welke je in je app.js gaat importeren.